live-examples/html-examples/input/css/button.css

```
@@ -7,24 +7,24 @@
   color: #fff;
   text-shadow: 1px 1px 1px #000;
   border-radius: 10px;
-  background-color: rgba(220, 0, 0, 1);
+  background-color: rgb(220 0 0 / 100%);
   background-image: linear-gradient(
     to top left,
-    rgba(0, 0, 0, 0.2),
-    rgba(0, 0, 0, 0.2) 30%,
-    rgba(0, 0, 0, 0)
+    rgb(0 0 0 / 20%),
+    rgb(0 0 0 / 20%) 30%,
+    rgb(0 0 0 / 20%)
```

Slight nit here, the original opacity was 0
```suggestion
    rgb(0 0 0 / 0%)
```